create.go

```diff
@@ -92,6 +93,19 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	// config.json provides a cgroups path that has to be used to create "tasks"
+	// and "cgroups.procs" files. Those files have to be filled with a PID, which
+	// is shim's in our case. This is mandatory to make sure there is no one
+	// else (like Docker) trying to create those files on our behalf. We want to
+	// know those files location so that we can remove them when delete is called.
+	if ociSpec.Linux.CgroupsPath == "" {
+		return fmt.Errorf("CgroupsPath should not be empty")
```

Comment:
@sameo - good catch. You're right - if it's specified, the runtime needs to honor it, but it's not mandatory:

- https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#control-groups